pkg/api/handlers/projects.go

```diff
@@ -623,10 +623,18 @@ func (h *ProjectsHandler) ListProjectThreads(req api.Context) error {
 		})
 	}
 
-	selector := kclient.MatchingFields{
-		"spec.project":          "false",
-		"spec.parentThreadName": projectThread.Name,
-		"spec.userUID":          req.User.GetUID(),
+	var selector kclient.ListOption
+	if req.UserIsAdmin() {
+		selector = kclient.MatchingFields{
+			"spec.project":          "false",
+			"spec.parentThreadName": projectThread.Name,
+		}
+	} else {
+		selector = kclient.MatchingFields{
+			"spec.project":          "false",
+			"spec.parentThreadName": projectThread.Name,
+			"spec.userUID":          req.User.GetUID(),
+		}
 	}
```

Comment:
nit: could you do something like this instead?

```suggestion
	selector := kclient.MatchingFields{
	    "spec.project":          "false",
	    "spec.parentThreadName": projectThread.Name,
	}
	if !req.UserIsAdmin() {
		selector["spec.userUID"] = req.User.GetUID()
	}
```

Reply:
Good call, done